Tests/SwiftParserTest/translated/TypealiasTests.swift

```diff
@@ -55,10 +53,8 @@ final class TypealiasTests: XCTestCase {
       typealias Foo21️⃣: Int
       """,
       diagnostics: [
-        // TODO: Old parser expected error on line 1: expected '=' in type alias declaration, Fix-It replacements: 15 - 16 = ' ='
-        DiagnosticSpec(message: "expected '=' and value in typealias declaration"),
-        DiagnosticSpec(message: "extraneous ': Int' at top level"),
-      ]
+        DiagnosticSpec(message: "expected '=' in typealias declaration"),
+      ], fixedSource: "typealias Foo2= Int"
```

Comment:
Any thoughts on running the basic format on this to add a space before/after the `=`?

Reply:
I explicitly didn’t want to do this because I wanted to match what the user wrote before. Basically trying to keep the applied change as minimal as possible.